git-filter-repo

```diff
@@ -3159,9 +3166,9 @@ class RepoFilter(object):
         blob.data = regex.sub(replacement, blob.data)
 
     if self._blob_callback:
-      self._blob_callback(blob, self.callback_metadata())
+      self._blob_callback(blob, self.callback_metadata(), self._callback_data)
 
-  def _tweak_commit(self, commit, aux_info):
+  def _tweak_commit(self, commit, aux_info, callback_data):
```

Comment:
Again, second argument; keep aux_info last.  But more importantly, why are you passing callback_data to this function, then ignoring it and using self._callback_data anyway?  What's the point in taking it as an extra parameter?